src/docs/asciidoc/reference/concepts/zetapush-npm-modules.adoc

```diff
@@ -1,2 +1,69 @@
 :relative-path: ../../
 include::{docdir}/variables.adoc[]
+
+ZetaPush is a *monorepo* that contains several *npm modules*. Each of them have a specific role.
+
+== @zetapush/cli
+
+Contains the *Command Line Interface* of ZetaPush.
+This is thanks to this module that you can launch `zeta run` or `zeta push`. +
+You can have more information in his <<reference#cli, dedicated section>>.
+
+== @zetapush/client
+
+When you <<reference#client_types, develop the front part of your application>>, i.e. the web pages displayed to your users.
+You use in particular a *ZetaPush client* to interact with your worker and the *ZetaPush cloud*.
+
+== @zetapush/cometd
+
+You don't use this module directly.
+It is used internally for the communication with the *ZetaPush cloud* via the protocol link:{cometd-url}[CometD].
```

Comment:
the protocol link:{cometd-url}[CometD] ->  the link:{cometd-url}[CometD] protocol